docs/src/pages/index.tsx

```diff
@@ -6,41 +6,274 @@ import clsx from "clsx";
+              <img className="toggle" src="img/burger-menu-right-svgrepo-com.svg" alt="menu toggle"/>
+              <img className="closee" src="img/close-svgrepo-com.svg" alt="menu close"/>
+            </button>
+          </div>
         </div>
-      </div>
-    </header>
+      </header>
   );
 }
 
 export default function Home(): JSX.Element {
   const { siteConfig } = useDocusaurusContext();
 
+const code = `Install necessary dependencies:;
+bash;
+pip install groundlight framegrab;
+Ask questions of your images:;
+
+pythonimport framegrabimport groundlight;
+# Initialize Groundlight client and create a Detector;
+gl = groundlight.Groundlight();
+det = gl.get_or_create_detector(name="doorway", query="Is the doorway open?");
+
+# Grab an image from a camera or video stream;
+grabber = framegrab.auto_discover()[0];
+image = grabber.grab_frame();
+
+
+# Process image and get a confident answer to the Detector's query;
+iq = gl.ask_confident(det, image);
+print(iq);
+<show output of printed iq here>`;
+
+  // Split the code into lines
+  const codeLines = code.split("\n");
+
   return (
     <Layout
       title={`${siteConfig.title}`}
       description="Computer Vision powered by Natural Language"
     >
       <HomepageHeader />
       <main>
-        <HomepageFeatures />
+        <section className="banner-section">
+          <div className="container">
+            <div className="banner-wrapper">
+              <h1 className="title">Build custom computer vision apps - faster & more reliably</h1>
+              <div className="content">
+                <p>With Groundlight’s Python SDK, you don’t need to be a machine learning scientist to develop your own
+                  computer vision application. Groundlight’s fully managed computer vision solution takes care of the ML so
+                  you can focus on building.</p>
+                <div className="button-wrapper">
+                  <a href="https://login.groundlight.ai/oauth2/register?tenantId=aad3d06b-ef57-454e-b952-91e9d3d347b1&client_id=ac8aac5d-c278-4c14-a549-e039f5ac54bb&nonce=&pendingIdPLinkId=&redirect_uri=https%3A%2F%2Fdashboard.groundlight.ai%2Fdevice-api%2Fauthz%2Fcallback&response_mode=&response_type=code&scope=openid%20profile%20email%20offline_access&state=&timezone=America%2FBogota&metaData.device.name=Windows%20Chrome&metaData.device.type=BROWSER&code_challenge=&code_challenge_method=&user_code=" className="cmn-button">Start building</a>
+                  <a href="https://code.groundlight.ai/python-sdk/docs/getting-started" className="cmn-button outline">See the docs</a>
+                </div>
+              </div>
+            </div>
+          </div>
+        </section>
+
+        <section className="feature-section">
+          <div className="container">
+            <div className="feature-wrapper">
+              <div className="feature-item">
+                <div className="icon">
+                  <img src="img/1.png" alt="img"/>
```

Comment:
"1.png" isnt a readable name for this icon. Can you rename 1.png, 2.png, 3.png to "brain1.png", "brain2.png", "brain3.png"?